### ISSUE TITLE
Cannot transition multiple Transitioning.View using animateNextTransition();

### ISSUE THREAD

**Issue by timbrandin** [software-mansion/react-native-reanimated#380]
Say you want to animate the background behind a switch-router, then you cannot syncronize them using animateNextTransition(), it will only animate the last one you called using this method.

i.e. in this case it will only animate t2 rather than both of them.
```
this.t1.current.animateNextTransition();
this.t2.current.animateNextTransition();
```

``` 
      <View style={{ flex: 1 }}>
        <Transitioning.View
          ref={this.t1}
          pointerEvents="none"
          transition={t1}
          style={StyleSheet.absoluteFill}
        >
          {state.index === 1 ? (
            <View key="b" style={[StyleSheet.absoluteFill, { backgroundColor: 'red' }]} />
          ) : (
            <View key="b" style={[StyleSheet.absoluteFill, { backgroundColor: 'green' }]} />
          )}
        </Transitioning.View>
        <Transitioning.View
          ref={this.t2}
          transition={t2}
          style={[styles.container, transitionViewStyle]}
        >
          <SceneView
            component={ChildComponent}
            navigation={descriptor.navigation}
            screenProps={screenProps}
          />
        </Transitioning.View>
      </View>
```

**Comment by DovletAmanov** [comment]
same issue +1

**Comment by DovletAmanov** [comment]
I am using setTimeout() for now, Try: 

```
this.t1.current.animateNextTransition();
setTimeout(() => {
      this.t2.current.animateNextTransition();
},200);
```

**Comment by adkenyon** [comment]
I moved the `Transition.View` higher in the tree. Obviously this may not work for everyone.

**Comment by jorgebaralt** [comment]
Any other solution for this?

**Comment by DaveClissold** [comment]
I'm also having this problem. Have tried moving the 2 Transitioning.Views into their own components and it will still only render t2 and triggering animateNextTransition() in a useEffect()

**Comment by Ailuue** [comment]
Really wish there was an update to this. Allowing for multiple transitions to be triggered from a single state change is a super common pattern.

**Comment by serfgy** [comment]
Is this slated for a confirmed issue or is this intended? Moving forward what should be the correct practice?

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.
 
We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
